cd/cd.yml

```diff
@@ -177,6 +184,14 @@ Resources:
                 Effect: Allow
                 Resource:
                   - !GetAtt CodePipelineFailurePollerFunction.Arn
+      Tags:
+        - Key: Project
+          Value: Infrastructure
+        - Key: "prx:cloudformation:stack-name"
+          Value: !Ref AWS::StackName
+        - Key: "prx:cloudformation:stack-id"
+          Value: !Ref AWS::StackId
+  CodePipelineFailurePollerFunction:
```

Comment:
What's with all these `CodePipelineFailurePollerFunction`s?

Reply:
Ugh. yeah looks like it